torch/backends/xeon/run_cpu.py

```diff
@@ -281,7 +281,19 @@ def add_lib_preload(self, lib_type):
                     break
         return lib_set or lib_find
 
-
+    
+    def is_numactl_available(self):
+        numactl_available = False
+        try:
+            cmd = ["numactl", "-C", "0", "-m", "0", "hostname"]
+            r = subprocess.run(cmd, env=os.environ)
```

Comment:
please redirect the output to /dev/null. We don't want to print result of this test command out.

Reply:
Thanks @jingxu10, done.